### ISSUE TITLE
Attempting to load this extension manually into Chrome is not working?

### ISSUE THREAD

**Issue by JayceTheBased** [voythas/shine-unofficial#40]
Hi, I'd been using this unofficial fork of the old extension for at least a couple years and been absolutely loving it. Since I reset my system, however, I can't seem to be able to get it working again with my new Chrome browser.

I'm sure this might be a dumb question, but when I'm trying to load this extension into Chrome it doesn't seem to work. I downloaded the code from here, unzipped it, and then clicked "load unpacked extension" in my chrome extensions and selected the "shine unofficial" folder. However, when I do that, I get an error that says "Manifest is not valid JSON. Line: 72, column: 1, Dictionary keys must be quoted."

I saw a guy saying that we could load this extension manually on another issue thread, so I feel like I must be missing something?

**Comment by dotvhs** [comment]
Here's .crx extension but as far as I know you will have to load it each time you restart your browser ;/ I didn't work on the solution yet. if you need to unpack crx just rename it to .zip.
[dlbccbcpelghmhkhmpefncahafgigkek_main.crx.zip](https://github.com/dotvhs/shine-unofficial/files/7742015/dlbccbcpelghmhkhmpefncahafgigkek_main.crx.zip)

**Comment by JayceTheBased** [comment]
> Here's .crx extension but as far as I know you will have to load it each time you restart your browser ;/ I didn't work on the solution yet. if you need to unpack crx just rename it to .zip. [dlbccbcpelghmhkhmpefncahafgigkek_main.crx.zip](https://github.com/dotvhs/shine-unofficial/files/7742015/dlbccbcpelghmhkhmpefncahafgigkek_main.crx.zip)

Thanks so much man! It looks like I may just have to reenable the extension in the chrome extension page every time I restart the browser, which isn't as bad as having to fully reload it. Thank you too for all the work you put into updating this in the past, I'm so glad that I can use it again at least somewhat for now! 

I'll echo the other guy saying that it would be great to have future updates, but it's up to you and your availibility to work on it of course.

**Comment by riotinprogress** [comment]
Thanks for that link. I thought I saved all of my extensions before moving my profile to a new computer but I guess not. Really hoping someone takes this over or at least updates it.

**Comment by scottrepreneur** [comment]
> Here's .crx extension but as far as I know you will have to load it each time you restart your browser

is it possible to push this so it shows up as a "release"? had to dig a bit to find this. when I try loading the full repo as unpacked I'm seeing some conflicts that didn't get resolved in [base.js](https://github.com/dotvhs/shine-unofficial/blob/master/base.js#L107) and [manifest.json](https://github.com/dotvhs/shine-unofficial/blob/master/manifest.json#L72)

Develop also appears to be in better shape per #34

**Comment by dotvhs** [comment]
> is it possible to push this so it shows up as a "release"?

You mean to publish on Chrome Webstore? I doubt there's hope for it. Google removed it from there because they didn't like basically everything (mostly embeds) which is a core Shine feature. I doubt it will ever make a return to Chrome Webstore. It's far more likely to be a userscript if I find enough time.

**Comment by scottrepreneur** [comment]
yeah totally get that. no i mean the [github releases](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) here. so it's kinda prominent when folks land at the repo. or add the link to the crx in the README?

<img width="380" alt="Screen Shot 2022-10-19 at 14 32 40" src="https://user-images.githubusercontent.com/1778380/196786642-4b486c18-c8e1-4ee0-bf2e-dd95bcb6129b.png">
